src/components/VueTypeaheadBootstrap.vue

```diff
@@ -33,6 +33,7 @@
         @input="handleInput($event.target.value)"
         @keydown.esc="handleEsc($event.target.value)"
         @keyup="$emit('keyup', $event)"
+        @matched-items="$emit('matched-items', $event)"
```

Comment:
let's call this `matched-items-count`?

```suggestion
        @matched-items-count="$emit('matched-items', $event)"
```